pkg/controller/workflow/reason.go

```diff
@@ -29,6 +29,8 @@ const (
 	ProjectIntegrationReady                  ConditionReason = "ProjectIntegrationReady"
 	ProjectPrivateEndpointIsNotReadyInAtlas  ConditionReason = "ProjectPrivateEndpointIsNotReadyInAtlas"
 	ProjectNetworkPeerIsNotReadyInAtlas      ConditionReason = "ProjectNetworkPeerIsNotReadyInAtlas"
+	ProjectEncryptionAtRestReady             ConditionReason = "ProjectEncryptionAtRestReady"
+	ProjectEncryptionAtRestFailed            ConditionReason = "ProjectEncryptionAtRestFailed"
```

Comment:
If status with such reason have "false" status, is Encryption at rest ready or not?

Reply:
Initially plan was to set this status if parameters are wrong and user has to change something but at this point I've changed this since so this status can be safely removed. Thanks for the catch 👍🏻